contactsummary/js/src/contact-summary.js

```diff
@@ -6,7 +6,5 @@ require.config({
 });
 
 require(['contact-summary/app'], function () {
-    document.addEventListener('contactsummaryLoad', function () {
```

Comment:
@AkA84 also if you have any idea in how to fix that without removing the listener that would be better .